client/extensions/woocommerce/app/orders/orders-list.js

```diff
@@ -143,36 +153,41 @@ class Orders extends Component {
 				</TableItem>
 			</TableRow>
 		);
-	}
+	};
 
 	renderOrderItems = () => {
 		const { orders, ordersLoading, translate } = this.props;
 
 		const headers = (
 			<TableRow isHeader>
-				<TableItem className="orders__table-name" isHeader>{ translate( 'Order' ) }</TableItem>
-				<TableItem className="orders__table-date" isHeader>{ translate( 'Date' ) }</TableItem>
-				<TableItem className="orders__table-status" isHeader>{ translate( 'Status' ) }</TableItem>
-				<TableItem className="orders__table-total" isHeader>{ translate( 'Total' ) }</TableItem>
+				<TableItem className="orders__table-name" isHeader>
+					{ translate( 'Order' ) }
```

Comment:
While we are making these changes, perhaps we should add context for this string, since Order is an overloaded term.